templates/interactive.config.j2

```diff
@@ -0,0 +1,47 @@
+[prompts]
+ADMIN_EMAIL = {{ dataverse.service_email }}
+GLASSFISH_ADMIN_USER = {{ dataverse.glassfish.adminuser }}
+GLASSFISH_ADMIN_PASSWORD = {{ dataverse.glassfish.adminpass }}
+GLASSFISH_DIRECTORY = {{ dataverse.glassfish.dir }}.
+GLASSFISH_HEAP = {{ dataverse.memheap }}
+GLASSFISH_USER = {{ dataverse.glassfish.user }}
+POSTGRES_ONLY =
+POSTGRES_ADMIN_PASSWORD = Postgres ADMIN password
+POSTGRES_SERVER = {{ dataverse.db.host }}
+POSTGRES_PORT = {{ dataverse.db.port }}
+POSTGRES_DATABASE = {{ dataverse.db.name }}
+POSTGRES_USER = {{ dataverse.db.user }}
+POSTGRES_PASSWORD = {{ dataverse.db.pass }}
+RSERVE_HOST = {{ rserve.host }}
+RSERVE_PORT = {{ rserve.port }}
+RSERVE_USER = {{ rserve.user }}
+RSERVE_PASSWORD = {{ rserve.pass }}
+HOST_DNS_ADDRESS = {{ public_hostname }}
+MAIL_SERVER = {{ dataverse.smtp }}
+SOLR_LOCATION = {{ solr_url }}
+TWORAVENS_LOCATION =
+DOI_USERNAME = {{ dataverse.doi.username }}
+DOI_PASSWORD = {{ dataverse.doi.password }}'
+DOI_BASEURL = {{ dataverse.doi.baseurl }}'
+DOI_MDCBASEURL = {{ dataverse.doi.mdcbaseurl }}'
+[comments]
+HOST_DNS_ADDRESS = :(enter numeric IP address, if FQDN is unavailable)
+GLASSFISH_USER = :This user will be running Glassfish service on your system.\n - If this is a dev. environment, this should be your own username; \n - In production, we suggest "glassfish" or another unprivileged user\n:
+GLASSFISH_DIRECTORY = 
+ADMIN_EMAIL = :\n(please enter a valid email address!) 
+MAIL_SERVER = 
+POSTGRES_SERVER = 
```

Comment:
I don't really understand what it means when there's no value? I assume it'll just use the default (localhost in this case, probably).

Reply:
hmmm. i think this was a partial template that got included by mistake. i'll remove it.